python/GangaDirac/Lib/Utilities/DiracUtilities.py

```diff
@@ -31,36 +29,78 @@
 def getDiracEnv(force=False):
     """
     Returns the dirac environment stored in a global dictionary by Ganga.
-    This is expected to be stored as some form of 'source'-able file on disk which can be used to get the printenv after sourcing
-    Once loaded and stored his is used for executing all DIRAC code in future
+    Once loaded and stored this is used for executing all DIRAC code in future
+
     Args:
         force (bool): This triggers a compulsory reload of the env from disk
     """
     global DIRAC_ENV
     with Dirac_Env_Lock:
-        if DIRAC_ENV == {} or force:
-            config_file = getConfig('DIRAC')['DiracEnvFile']
-            if not os.path.exists(config_file):
-                absolute_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '../../..', config_file)
+        if not DIRAC_ENV or force:
```

Comment:
So you're suggesting that we remove this argument? I think I agree. It won't necessarily do what is expected as it will only clear the in-memory cache, not the disk-cache. Nowhere uses in the code so it's safe to remove.

I will remove it in another commit incoming.